tests/release.yaml

```
@@ -13,4 +13,6 @@ releases:
       listener:
         operatorVersion: 0.0.0-dev
       airflow:
+        operatorVersion: 0.0.0-pr404
```

This can now be selected in Jenkins again.

```suggestion
        operatorVersion: 0.0.0-dev
```